cves/CVE-2011-3192.yml

```diff
@@ -52,14 +56,10 @@ fixes:
      note:
```

Comment:
I would definitely label this one as the fix instead of leaving the note field blank.

Reply:
Labeled